src/frontend/apps/desk/src/features/mail-domains/access-management/components/ChooseRole.tsx

```diff
@@ -1,65 +1,60 @@
 import { Radio, RadioGroup } from '@openfun/cunningham-react';
 import { useTranslation } from 'react-i18next';
 
-import { Role } from '../../domains';
+import { ORDERED_ROLES, Role } from '../../domains';
 
 interface ChooseRoleProps {
   availableRoles: Role[];
+  roleForSelectedUser: Role;
```

Comment:
Why not calling it `roleAccess` as from its type ?
https://github.com/numerique-gouv/people/blob/f116f7805f5a22647a8451b3d051cf4052340bc3/src/frontend/apps/desk/src/features/mail-domains/access-management/types.ts#L9-L10